src/pyfmi/fmi3.pxd

```diff
@@ -52,6 +52,13 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
     cdef FMIL.fmi_version_enu_t     _version
     cdef FMIL.size_t _nEventIndicators  # format with snake case?
     cdef FMIL.size_t _nContinuousStates # format with snake case?
+    cdef FMIL3.fmi3_boolean_t _event_info_new_discrete_states_needed
+    cdef FMIL3.fmi3_boolean_t _event_info_terminate_simulation
+    cdef FMIL3.fmi3_boolean_t _event_info_nominals_of_continuous_states_changed
+    cdef FMIL3.fmi3_boolean_t _event_info_values_of_continuous_states_changed
+    cdef FMIL3.fmi3_boolean_t _event_info_next_event_time_defined
+    cdef FMIL3.fmi3_float64_t _event_info_next_event_time
+    cdef public FMI3EventInfo _eventInfo
```

Comment:
Should we really have `public` and start the name with an underscore? I guess this is due to legacy reasons but is it too much to change?